game/rooms/room15/esc/right_exit.esc

```diff
@@ -0,0 +1,3 @@
+:exit_scene
+#set_sound_state _sound res://game/sfx/sounds/doorOpen_2.ogg false
```

Comment:
This file can be removed when there's only commented out code.

Reply:
Done